hack/podman-commands.sh

```diff
@@ -0,0 +1,40 @@
+man ./docs/podman-image.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-image.man
+echo diff -B -u /tmp/podman-image.cmd /tmp/podman-image.man
+diff -B -u /tmp/podman-image.cmd /tmp/podman-image.man
+
+./bin/podman container --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-container.cmd
+man docs/podman-container.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-container.man
+echo diff -B -u /tmp/podman-container.cmd /tmp/podman-container.man
+diff -B -u /tmp/podman-container.cmd /tmp/podman-container.man
+
+./bin/podman system --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-system.cmd
+man docs/podman-system.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-system.man
+echo diff -B -u /tmp/podman-system.cmd /tmp/podman-system.man
+diff -B -u /tmp/podman-system.cmd /tmp/podman-system.man
+
+./bin/podman play --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-play.cmd
+man docs/podman-play.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-play.man
+echo diff -B -u /tmp/podman-play.cmd /tmp/podman-play.man
+diff -B -u /tmp/podman-play.cmd /tmp/podman-play.man
+
+./bin/podman generate --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-generate.cmd
+man docs/podman-generate.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-generate.man
+echo diff -B -u /tmp/podman-generate.cmd /tmp/podman-generate.man
+diff -B -u /tmp/podman-generate.cmd /tmp/podman-generate.man
+
+./bin/podman pod --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-pod.cmd
+man docs/podman-pod.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-pod.man
+echo diff -B -u /tmp/podman-pod.cmd /tmp/podman-pod.man
+diff -B -u /tmp/podman-pod.cmd /tmp/podman-pod.man
+
+./bin/podman volume --help | sed -n -e '0,/Available Commands/d' -e '/^Flags/q;p' | sed '/^$/d' | awk '{ print $1 }' > /tmp/podman-volume.cmd
+man docs/podman-volume.1 | sed -n -Ee '0,/COMMANDS/d'  -e 's/^[[:space:]]*//' -e '/^SEE ALSO/q;p' | grep podman | cut -f1 -d' ' | sed 's/^.//' > /tmp/podman-volume.man
+echo diff -B -u /tmp/podman-volume.cmd /tmp/podman-volume.man
+diff -B -u /tmp/podman-volume.cmd /tmp/podman-volume.man
```

Comment:
pretty nifty chunk of sed and awk here....

Reply:
Yes, I am sure @nalind Would have done a better job but it gets the job done.
Want to make this a blocker when you call make validate, to make sure we don't have regressions in the future.

Reply:
For a blocker, as a later exercise, would you consider the friendly variation I submitted as a PR to your PR? That way there's less duplication and less hardcoding: if/when podman gains a new subcommand, my variation will pick it up automatically.

Reply:
Sure.  I'll look at it.  My only concern is the script is a little fragile.